var/spack/repos/builtin/packages/isescan/package.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Isescan(Package):
+    """A python pipeline to identify IS (Insertion Sequence) elements in
+       genome and metagenome"""
+
+    homepage = "https://github.com/xiezhq/ISEScan"
+    url      = "https://github.com/xiezhq/ISEScan/archive/v1.7.2.1.tar.gz"
+
+    version('1.7.2.1', sha256='b971a3e86a8cddaa4bcd520ba9e75425bbe93190466f81a3791ae0cb4baf5e5d')
+
+    depends_on('python@3.3.3:', type='run')
+    depends_on('py-numpy@1.8.0:', type='run')
+    depends_on('py-scipy@0.13.1:', type='run')
+    depends_on('py-biopython@1.62:', type='run')
+    depends_on('py-fastcluster', type='run')
+    depends_on('py-argparse', type='run')
+    depends_on('blast-plus@2.2.31:', type='run')
+    depends_on('fraggenescan@1.30:', type='run')
+    depends_on('hmmer@3.1b2:', type='run')
+
+    def setup_run_environment(self, env):
+        env.prepend_path('PATH', self.prefix)
+        env.prepend_path('LD_LIBRARY_PATH',
+                         join_path(self.prefix, 'ssw201507'))
+
+    def install(self, spec, prefix):
+        # build bundled SSW library
+        with working_dir('ssw201507'):
+            which('gcc')(
+                '-O3', '-pipe', '-fPIC', '-shared', '-rdynamic',
+                '-o', 'libssw.so', 'ssw.c', 'ssw.h',
+            )
+
+        # set paths to required programs
+        blast_pfx = self.spec['blast-plus'].prefix.bin
+        blastn_path = join_path(blast_pfx, 'blastn')
```

Comment:
```suggestion
        blastn_path = blast_pfx.blastn
```
Same for other lines